### ISSUE TITLE
[Feature]: Allow to pass backend type and backend options to `dstack config` as arguments

### ISSUE THREAD

**Issue by peterschmidt85** [dstackai/dstack#225]
### Problem

1. Hub will offer a code snippet to configure the backend in one command.
2. Sometimes, it's easier to pass certain backend parameters as arguments, e.g. file paths, because then you can rely on file name completion.

See both examples below.

### Solution

Usage: 

```shell
dstack config [BACKEND] [ARGS]

BACKEND - {aws|gcp|azure|hub}
ARGS - the arguments supported by the chosen backend type
```

If any of the backend arguments are not passed, the CLI will ask it in the interactive mode.

Example 1:

```shell
dstack config hub --url URL --token TOKEN
```

Example 2:

```shell
dstack config gcp --key-file SERVICE_ACCOUNT_KEY_JSON_FILE 
```

The `--help` argument should also work:

```shell
dstack config hub --help
```

The principle should be the same as for provider arguments support in `dstack run`.

### Benefit

-

### Alternatives

-

### Would you like to help contributing this feature?

No

**Comment by r4victor** [comment]
No longer relevant after #338
